src/Components/MainTableView/TableContent.tsx

```diff
@@ -59,38 +56,37 @@ class TableContent extends React.Component<{
     selectedTableName: string, 
     selectedTableType: TableType, 
     contentData: Array<any>, 
-    tableTotal: number,
-    tableAttributesInfo?: TableAttributesInfo, 
+    totalNumOfTuples: number,
+    currentPageNumber: number,
+    maxPageNumber: number,
+    tuplePerPage: number,
+    tableAttributesInfo?: TableAttributesInfo,
+    setPageNumber: any,
```

Comment:
Specifying a type of `any` really defeats the purpose of static typing and most statically typed languages don't usually offer this. I know it is used extensively everywhere already but we should avoid declaring any new variables or attributes with `any` type. If a variable has multiple type requirements, look into union and intersect types. If we 'need' to use `any` type, I'd like to here the justification.

Reply:
I was going to leave this for when I go through and document stuff. Right now I am just trying to keep it lose to avoid type errors and type script complain left and right.

Reply:
I have this fixed in a different branch that handles documentation. Thus to avoid merge conflict I perfer not to change it on this pull request.